ietf/status/migrations/0001_initial.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 4.2.13 on 2024-06-24 04:24
```

Comment:
Replace the Generated by with the trust copyright line

Reply:
Also, since this is a new app coming in, delete the migrations and regenerate them so that there's only a 0001_initial.